staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/noderegistrar.go

```diff
@@ -30,17 +30,15 @@ type nodeRegistrar struct {
 }
 
 // startRegistrar returns a running instance.
-//
-// The context is only used for additional values, cancellation is ignored.
-func startRegistrar(valueCtx context.Context, grpcVerbosity int, interceptors []grpc.UnaryServerInterceptor, streamInterceptors []grpc.StreamServerInterceptor, driverName string, supportedServices []string, endpoint string, pluginRegistrationEndpoint endpoint) (*nodeRegistrar, error) {
+func startRegistrar(logger klog.Logger, grpcVerbosity int, interceptors []grpc.UnaryServerInterceptor, streamInterceptors []grpc.StreamServerInterceptor, driverName string, supportedServices []string, pluginEndpoint string, pluginRegistrationEndpoint endpoint) (*nodeRegistrar, error) {
```

Comment:
Question: why is `pluginEndpoint` not an `endpoint` type? Does this mean that the argument should be renamed or the type changed?

Reply:
`registerapi.PluginInfo` as an `Endpoint string` field. This name here reflects that more than the rest of the code where it normally would be called "socket path". I've switched to that.